lib/src/models/account.dart

```diff
@@ -1,65 +1,77 @@
 import 'emails.dart';
 import 'profile.dart';
-import 'session_info.dart';
 
 /// This class represents an account.
 class Account {
   /// The default constructor.
   Account({
     required this.emails,
-    this.created,
     this.createdTimestamp,
     this.isActive,
     this.isRegistered,
     this.isVerified,
     this.lastLoginTimestamp,
     this.lastUpdatedTimestamp,
     this.loginProvider,
-    this.oldestUpdateTimestamp,
+    this.oldestDataUpdateTimestamp,
```

Comment:
I reverted the rename here, just to be consistent with the naming of the actual data.